src/components/CompareResults/loader.ts

```diff
@@ -66,6 +55,67 @@ function checkValues({
     );
   }
 
+  //check the new repos and revs first for the case of compare over time component
+
+  if (newRevs.length !== newRepos.length) {
+    throw new Error(
+      'There should be as many "newRepo" parameters as there are "newRev" parameters.',
+    );
+  }
+
+  //if interval is not null, return values for time range component
+  if (interval !== null) {
+    if (interval === null) {
```

Comment:
ignore this line; I removed it in commit #4 since of course it doesn't make sense :P